dataland-frontend/src/types/HighImpactClimateSectors.ts

```diff
@@ -0,0 +1,23 @@
+export enum HighImpactClimateSectorsNaceCodes {
+  "NaceCodeAInGWh" = "A",
+  "NaceCodeBInGWh" = "B",
+  "NaceCodeCInGWh" = "C",
+  "NaceCodeDInGWh" = "D",
+  "NaceCodeEInGWh" = "E",
+  "NaceCodeFInGWh" = "F",
+  "NaceCodeGInGWh" = "G",
+  "NaceCodeHInGWh" = "H",
+  "NaceCodeLInGWh" = "L",
+}
+
+export enum HighImpactClimateSectorsKeys {
```

Comment:
Instead this should state something like the following:

![image](https://github.com/d-fine/Dataland/assets/35459207/c9732ecf-a139-4a83-840a-4ca269c48bb9)

Reply:
<img width="205" alt="nace" src="https://github.com/d-fine/Dataland/assets/113936582/25d91c10-5ce3-4ba3-8a4f-d7616021839a">
Would this as a label be fine? Retrieving these would be automated. If not I can just change the enum values.

Reply:
yes right

Reply:
Yes - that would be acceptable

Reply:
changed